tutorials/fiori-ios-scpms-starter-mission-03/fiori-ios-scpms-starter-mission-03.md

```diff
@@ -133,9 +133,22 @@ The Product List is a Table View Controller which means the structure is similar
 
 2. Now we will add parts of the class properties we already have used in the Overview Table View Controller. Implement the following lines of code at the top of the class:
 
+    **For Online OData**
+
+    Add the following import statement to your class:
+
+    ```Swift
+    import SAPOData
+
+    ```
+
+    Implement the following lines of code directly below the logger instance as class properties:
+
     ```Swift
+    /// First retrieve the destinations your app can talk to from the AppParameters.
     let destinations = FileConfigurationProvider("AppParameters").provideConfiguration().configuration["Destinations"] as! NSDictionary
 
+    /// Create a computed property that uses the OnboardingSessionManager to retrieve the onboarding session and uses the destinations dictionary to pull the correct destination. Of course we only have one destination here. Handle the errors in case the OData controller is nil. We are using the AlertHelper to display an AlertDialogue to the user in case of an error. The AlertHelper is a utils class provided through the iOS Assistant.
```

Comment:
Please don't say "we", say "you" instead as described in the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqgno1sd).